src/config.h

```diff
@@ -65,9 +65,17 @@ void config_add_loader(const char* section, config_loader loader);
 bool config_to_bool(const char* text, bool* flag);
 
 /**
- * Convert text value to ARGB color.
+ * Convert text value to ARGB color, with a constant max value for alpha.
  * @param text text to convert
  * @param color output variable
  * @return false if text has invalid format
  */
 bool config_to_color(const char* text, argb_t* color);
+
+/**
+ * Convert text value to ARGB color which may be any alpha.
+ * @param text text to convert
+ * @param color output variable
+ * @return false if text has invalid format
+ */
```

Comment:
I don't think we need two similar functions to parse color.

Reply:
I may be mistaken about the cause, but if the `config_to_translucent_color` is used for all colors then I get some weird effects on the overlay's font. I imagine this is because the alpha value is used in `text.c` for special meanings (absence/presence of shadow), unfortunately mirroring the original behavior of giving an alpha of `0x00` when none is given would result in background colors not working correctly when given an opaque color, as it would translate to being transparent (alpha of 0x00), rather than opaque (alpha of 0xff) like in the new function.

Could easily be paramterized, but behavior must differ between parsing `#RRGGBB` and `#RRGGBBAA`. I didn't parameterize it in order to minimize the PR's changes.